docs/next.config.js

```diff
@@ -41,7 +41,8 @@ module.exports = {
     REACT_STRICT_MODE: reactStrictMode,
     // Set by Netlify
     GRID_EXPERIMENTAL_ENABLED: process.env.PULL_REQUEST === 'false' ? 'false' : 'true',
-    SOURCE_CODE_ROOT_URL: 'https://github.com/mui-org/material-ui-x/blob/next',
+    // #default-branch-switch
```

Comment:
This is a breadcrumb we leave so the next time we change the name of the active branch, we can search it and replace all the instances blindly.